server/migrations/20190111130729-create-articleLikesDislike.js

```diff
@@ -0,0 +1,33 @@
+export default {
```

Comment:
why do same files appear twice and also could this be the reason why the number of files changes has increased.
Travis build for the branch is also failing

Reply:
You're right. It's due to Github being case-insensitive to file renaming. It's been sorted now thanks.